.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=true
```

Comment:
Nitpick: `package-lock` is enabled by default, it may be better to remove `.npmrc` completely. Just FYI, no change is needed.

Reply:
I'm aware but as mentioned in a different PR by @raymondfeng some developers like him have it disabled globally by default -- this was intentional :)

Reply:
Good point! I have package-lock disabled globally too.